Cargo.toml

```diff
@@ -10,12 +10,16 @@ license = "MIT OR Apache-2.0"
 keywords = ["TES3", "Morrowind"]
 categories = ["Game development"]
 
+[lib]
```

Comment:
this might be unnecessary. Can't remember adding it, must've done so in the other repo